src/main/java/com/questhelper/helpers/quests/meatandgreet/MeatAndGreet.java

```diff
@@ -0,0 +1,486 @@
+		headWestAndEnterTheDungeon = new ObjectStep(this, ObjectID.WOLF_DEN_54692, new WorldPoint(1498, 3132, 0), "Head west from the farm and enter the Wolf Den, prepared to kill the Dire Wolf Alpha.", combatGear, food);
+		var enterWolfDenAndKillTheDireWolfAlpha = new ConditionalStep(this, headWestAndEnterTheDungeon);
+		// enterWolfDenAndKillTheDireWolfAlpha
+		killDireWolfAlpha = new NpcStep(this, NpcID.DIRE_WOLF_ALPHA, "Kill the Dire Wolf Alpha. Use Protect from Melee to avoid most damage. His pups deal ranged damage.", combatGear, food);
+		// TODO: Add check for "need to kill wolf"
+		// killed wolf: 11184 2->3
+		var needToKillWolf = new VarbitRequirement(11184, 2);
+		var exitWolfDen = new ObjectStep(this, ObjectID.CAVE_54691, new WorldPoint(1488, 9502, 1), "Exit the wolf den, then head back to Alba in the farmhouse west of Civitas illa Fortis.");
+		// 13092 0->100
+		// 13093 0->100
+		// 13094 0->100
+		// 13095 0->100
+		returnToAlba = new NpcStep(this, NpcID.ALBA, new WorldPoint(1587, 3126, 0), "Return to Alba in the farmhouse west of Civitas illa Fortis to tell them about the Dire Wolf Alpha's demise.");
+		returnToAlba.addSubSteps(exitWolfDen);
+		var needToReturnToAlba = new VarbitRequirement(11184, 3);
+		// 11184 3->4 after returning to alba
+		// TODO: Confirm the return to alba step works
+		solveSupplyChainIssues = new ConditionalStep(this, returnToAlba);
+		solveSupplyChainIssues.addStep(pinPadOpen, enterCodeWrapper);
+		solveSupplyChainIssues.addStep(needToSendSpicesFromSpiceMerchant, talkToSpiceMerchantInBazaar);
+		solveSupplyChainIssues.addStep(and(needToReturnToAlba, inWolfDen), exitWolfDen);
+		solveSupplyChainIssues.addStep(and(needToKillWolf, inWolfDen), killDireWolfAlpha);
+		solveSupplyChainIssues.addStep(needToKillWolf, enterWolfDenAndKillTheDireWolfAlpha);
+		solveSupplyChainIssues.addStep(needToTalkToAlbaAboutWolves, talkToAlba);
+
+		/// 6
+		// Can save 50% of the time by teleporting to Civitas illa Fortis & running south-east, or using a Quetzal whistle to (?), but I don't have a whistle!
+		returnToEmelio = new NpcStep(this, NpcID.EMELIO, new WorldPoint(1753, 3074, 0), "Return to Emelio in Outer Fortis at the south-eastern entrance of Civitas illa Fortis to find the perfect ratio of ingredients for the kebab.");
+		returnToEmelio.addTeleport(civitasIllaFortisTeleport);
+
+		/// 8
+		var giveRecipeToEmelio = new NpcStep(this, NpcID.EMELIO, new WorldPoint(1753, 3074, 0), "Tell Emelio the recipe is perfect.");
+		giveRecipeToEmelio.addDialogStep("I think so.");
+		giveRecipeToEmelio.addDialogStep("I think that's perfect!");
+		giveRecipeToEmelio.addDialogStep("Sounds good to me.");
+		var adjustRecipe = new NpcStep(this, NpcID.EMELIO, new WorldPoint(1753, 3074, 0), "Adjust the recipe with Emelio. Four portions of meat, two portions of salad, one portion of spice, and three portions of sauce.");
```

Comment:
I was put onto this step after returning to him, and needed to select 'I think so.' here to go to adjusting the proportions.

Reply:
Fixed in [`54ebeec` (#1778)](https://github.com/Zoinkwiz/quest-helper/pull/1778/commits/54ebeec5dec6b7ad5294cc8ad828ac2659875980)